rakelib/form526.rake

```diff
@@ -226,4 +226,127 @@ namespace :form526 do
     end
     puts "reuploaded files for #{form_submissions.count} submissions"
   end
+
+  desc 'Convert SIP data to camel case and fix checkboxes [/export/path.csv, ids]'
+  task :convert_sip_data, [:csv_path] => :environment do |_, args|
+    raise 'No CSV path provided' unless args[:csv_path]
+
+    ids = args.extras || []
+
+    def to_olivebranch_case(val)
+      OliveBranch::Transformations.transform(
+        val,
+        OliveBranch::Transformations.method(:camelize)
+      )
+    end
+
+    def un_camel_va_keys!(hash)
+      json = hash.to_json
+      # rubocop:disable Style/PerlBackrefs
+      # gsub with a block explicitly sets backrefs correctly https://ruby-doc.org/core-2.6.6/String.html#method-i-gsub
+      json.gsub!(OliveBranch::Middleware::VA_KEY_REGEX) do
+        key = $1
+        "#{key.gsub('VA', 'Va')}:"
+      end
+      JSON.parse(json)
+      # rubocop:enable Style/PerlBackrefs
+    end
```

Comment:
might be nice to reference where this is coming from:
https://github.com/department-of-veterans-affairs/vets-api/blob/af4112bbadff92485a90a479c0852b8aac9a2c23/spec/lib/olive_branch_patch_spec.rb
or use the original as you are here:
https://github.com/department-of-veterans-affairs/vets-api/pull/5651/files#diff-0f1b0208c79d97ef65a1049c67fc2ab70333e8eb2bf54ff0b8d04b0640b0aeeaR337